.github/workflows/build-with-bal-test-graalvm.yml

```diff
@@ -0,0 +1,53 @@
+name: GraalVM Check
+
+on:
+  schedule:
+    - cron: "30 18 * * *"
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, labeled]
+
+concurrency:
+  group: graalvm-check
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check')
+
+    steps:
```

Comment:
Can't we use the GraalVM connector template for this workflow file?

Reply:
Fixed with [69492b9e569069b5ab740b394085504a601f2728](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/69492b9e569069b5ab740b394085504a601f2728)